zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -216,13 +216,13 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
         let no_callee_code = is_empty_code_hash.expr() + callee_not_exists.expr();
 
         // TODO: And not precompile
-        cb.condition(not::expr(tx_is_create.expr()), |cb| {
-            cb.account_read(
-                tx_callee_address.to_word(),
-                AccountFieldTag::CodeHash,
-                code_hash.to_word(),
-            ); // rwc_delta += 1
-        });
+        // i think this needs to be removed....
```

Comment:
Do you mean removing the "TODO" in line 218?  If that's the case, I agree it can be removed!